sdks/java/extensions/sketching/src/main/java/org/apache/beam/sdk/extensions/sketching/ApproximateDistinct.java

```diff
@@ -200,6 +200,9 @@
  *
  * }</pre>
  *
+ * Consider using {@code HllCount} in the {@code zetasketch} extension module if you need to create
+ * sketches with format compatible with Google Cloud BigQuery.
```

Comment:
Could you please a code example here about how to use your transform instead of `ApprocimateDistinct` and attach your design doc, especially `Comparison with existing Beam transforms: ApproximateUnique & ApproximateDistinct` section.

Reply:
Done.